tests/test_isr.py

```diff
@@ -0,0 +1,140 @@
+#
+# This file is part of obs_cfht.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (http://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+import os
+import unittest
+import warnings
+
+import lsst.afw.image as afwImage
+import lsst.utils.tests
+
+from lsst.obs.cfht import MegaPrime
+from lsst.obs.cfht.cfhtIsrTask import CfhtIsrTask
+
+
+class CfhtIsrTestCase(lsst.utils.tests.TestCase):
+    """Test for the CFHT IsrTask wrapper."""
+
+    def setUp(self):
```

Comment:
Since there's only one test case, it doesn't make any difference now, but if you foresee other test cases making use of the `exposure` and `camera` attributes, make it a `setUpClass` method so that it doesn't have to read in the images for every test case.